includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -124,24 +124,26 @@ private function getPageUrl( $pageTitle, $namespace, $query = '' ) {
 	private function getSearchData() {
 		$isCorporatePage = WikiaPageType::isCorporatePage( $this->productInstanceId );
 
-		if ( $this->product === static::PRODUCT_FANDOMS ) {
+		if ( $this->product === static::PRODUCT_FANDOMS && $this->lang === static::DEFAULT_LANG ) {
 			$searchUrl = '/';
 			$searchPlaceholderKey = 'global-navigation-search-placeholder-fandom';
+			$searchParamName = 's';
 		} else {
-			if ( $isCorporatePage ) {
+			if ( $isCorporatePage || $this->product === static::PRODUCT_FANDOMS ) {
 				$searchUrl = $this->getCorporatePageSearchUrl();
 				$searchPlaceholderKey = 'global-navigation-search-placeholder-wikis';
 			} else {
 				$searchUrl = $this->getPageUrl( 'Search', NS_SPECIAL, [ 'fulltext' => 'Search' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Search" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)